src/components/navbar/Eventhighlight.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
Please make a seperate folder for these components, instead of putting in navbar folder.